cobalt/persistent_storage/persistent_settings.h

```diff
@@ -101,6 +103,18 @@ class PersistentSettings : public base::MessageLoop::DestructionObserver {
     return pref_store_;
   }
 
+  void commit_pending_write(bool blocking) {
```

Comment:
Maybe better put this function implementation in persistent_settings.cc rather than here?

Reply:
ack

Reply:
updated in a following PR